### PR TITLE
Added test TEST-40373,TEST-40374

### PR DESCRIPTION
**Pull request by pragamj**
Signed-off-by: pragam <pragam.jain@seagate.com>

# Problem Statement
- Provisioner backlog test-cases  TEST-40373,TEST-40374

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
[TEST-40373_TestProvPodsDeployment_20220527044709.log](https://github.com/Seagate/cortx-test/files/8786312/TEST-40373_TestProvPodsDeployment_20220527044709.log)
[TEST-40374_TestProvPodsDeployment_20220527043649.log](https://github.com/Seagate/cortx-test/files/8786313/TEST-40374_TestProvPodsDeployment_20220527043649.log)
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
[collection_log.log](https://github.com/Seagate/cortx-test/files/8786314/collection_log.log)
# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: libs/motr/motr_core_k8s_lib.py [hunk 1]
```diff
@@ -360,13 +360,16 @@ def diff_cmd(self, file1, file2, node):
         :file2: second file
         :node: compare files on which node
         """
-
+        diff_utils_install = common_cmd.INSTALL_DIFF_UTIL
         cmd = common_cmd.DIFF.format(file1, file2)
-        resp = self.node_obj.send_k8s_cmd(operation="exec", pod=self.node_pod_dict[node],
```
Comment (pranav-sahasrabudhe) [inline libs/motr/motr_core_k8s_lib.py:365]: In general I can see a lot of optimization can be done at `commands.py`  but will differ that review for sometime later in the housekeeping.

**Review by pranav-sahasrabudhe — APPROVED** [review]
Approved. Check the comment for the future optimization.

File: libs/motr/motr_core_k8s_lib.py (hunk 1 above)
Comment (pragamj) [reply libs/motr/motr_core_k8s_lib.py:365]: sure.

Comment (pragamj) [reply libs/motr/motr_core_k8s_lib.py:365]: sure.

File: commons/commands.py [hunk 2]
```diff
@@ -400,6 +400,7 @@
 M0CAT = "m0cat -l {} -H {} -P {} -p {} -s {} -c {} -o {} -L {} {}"
 M0UNLINK = "m0unlink -l {} -H {} -P {} -p {} -o {} -L {}"
 M0KV = "m0kv -l {} -h {} -f {} -p {} {}"
+INSTALL_DIFF_UTIL = "yum install diffutils -y"
```
Comment (ketanarlulkar) [inline commons/commands.py:403]: Use CMD_INSTALL_TOOL defined above line 375

Comment (pragamj) [reply commons/commands.py:403]: done

**Review by ravindrachoudhari — APPROVED** [review]

**Review by ketanarlulkar — APPROVED** [review]

**Review by gaurkrgaurav — APPROVED** [review]

**Pull request merged**